api/PerformanceLongTaskTiming.json

```diff
@@ -97,6 +97,53 @@
             "deprecated": false
           }
         }
+      },
+      "toJSON": {
```

Comment:
Since the test tests the prototype, can you verify this isn't inherited and a false positive?

Reply:
Just did some testing in Chrome in BrowserStack, it looks like this is accurate!

Reply:
Great, thank you!